### PR TITLE
Python3 msg/srv changes

### PR DESCRIPTION
**Pull request by FuriousJulius**
Pulled the service and msg type out of the appctl package into appctl_msg_defs.  @minshallj did a bit of sleuthing and thinks we still use the msg/srv in our portal state_tracker node so we didn't just remove the srv/msg.  Removed unneeded deps in our Dockerfile.

### CODE REVIEW

File: appctl_msg_defs/package.xml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  <maintainer email="lg@endpoint.com">End Point Corp</maintainer>
+
+  <author email="wojtek@endpoint.com">Wojciech Ziniewicz</author>
+
+  <url type="website">https://liquidgalaxy.endpoint.com/</url>
+
+  <license>Apache 2.0</license>
+
+  <buildtool_depend>catkin</buildtool_depend>
+  <build_depend>rospy</build_depend>
```
Comment (mvollrath) [inline appctl_msg_defs/package.xml]: This package probably just needs catkin and message gen/runtime.  Refer to [the std_msgs package](https://github.com/ros/std_msgs/blob/kinetic-devel/package.xml) which also defines messages with no msg dependencies.  Same for CMakeLists.txt

File: appctl_msg_defs/srv/Query.srv [hunk 2]
```diff
@@ -0,0 +1,2 @@
+---
+string mode
```
Comment (minshallj) [inline appctl_msg_defs/srv/Query.srv]: I see this and `Mode.msg` added, but I don't see them being removed from `appctl/`

File: appctl_msg_defs/package.xml [hunk 3]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
+<package>
+  <name>appctl_msg_defs</name>
+  <version>0.0.1</version>
+  <description>Support software for mode switching</description>
+
+  <maintainer email="lg@endpoint.com">End Point Corp</maintainer>
+
+  <author email="wojtek@endpoint.com">Wojciech Ziniewicz</author>
```
[the remark this answers is not in the document]
Comment (FuriousJulius) [reply appctl_msg_defs/package.xml]: Fixed :+1:

File: appctl_msg_defs/package.xml (hunk 1 above)
Comment (FuriousJulius) [reply appctl_msg_defs/package.xml]: Cool, thanks for the example.  Makes sense.

File: appctl_msg_defs/srv/Query.srv (hunk 2 above)
Comment (FuriousJulius) [reply appctl_msg_defs/srv/Query.srv]: :skull: whoops.  Fixed :+1: :taco:

File: appctl_msg_defs/package.xml [hunk 4]
```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
+<package>
+  <name>appctl_msg_defs</name>
+  <version>0.0.1</version>
+  <description>Msg and service defs for appctl pkg</description>
+
+  <maintainer email="lg@endpoint.com">End Point Corp</maintainer>
+
+  <author email="matt@endpoint.com">Matt Vollrath</author>
```
Comment (mvollrath) [inline appctl_msg_defs/package.xml]: That's not how you spell William Plaut..

**Pull request merged**